Dokan/Dokan/Domain/Sources/Domain/Domain.swift

```diff
@@ -0,0 +1,13 @@
+
+
+struct User :Decodable{
+    let id:Int?
+    let username, firstname, lastname, email: String?
+    let imageUrl: String?
+
+}
+
+protocol DomainRepository{
```

Comment:
```suggestion
protocol UserRepository {
```
And create new file named `UserRepository.swift` and new group under name `Repository`